app/controllers/api/v1/bikes_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Api::V1::BikesController < ApplicationController
+  # GET /bikes
+  def index
+    @bikes = Bike.all
+
+    render json: @bikes
+  end
+
+  # GET /bikes/1
+  def show
+    render json: @bike
+  end
```

Comment:
- It looks like there's something to improve on show methods. This is what I see when I try this:
<img width="494" alt="image" src="https://user-images.githubusercontent.com/20563252/213588689-e0e1e565-22fb-498c-81c1-83844eb679dc.png">

I kindly invite you to first, find the bike and then share the info for this. 🐶

Reply:
hey @vichuge,

yeah it just because the detabase is empty for some reason you can see now ..

![Screenshot from 2023-01-20 05-46-09](https://user-images.githubusercontent.com/104095543/213594056-215ee919-6d16-4e83-9a34-c6412b8193e3.png)

you could've run the seed file it would work. well somehow let me run it now.